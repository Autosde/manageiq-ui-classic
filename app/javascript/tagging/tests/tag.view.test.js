import React from 'react';
import TagView from '../components/tagView';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

const setTags = [{ tagCategory: 'animal', tagValue: 'pig' }, { tagCategory: 'planet', tagValue: 'Earth' }];
function onDelete(x) {
  return x;
}
describe('Tag view', () => {
  it('match snapshot', () => {
    const component = renderer.create(<TagView setTags={setTags} onTagDeleteClick={onDelete} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
