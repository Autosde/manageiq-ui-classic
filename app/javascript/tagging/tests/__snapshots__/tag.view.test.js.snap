// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some dummy test 1`] = `
<ul
  className="list-inline"
>
  <li>
    <span
      className="label label-info"
    >
      animal
      : 
      pig
      <a
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
          className="pficon pficon-close"
        />
        <span
          className="sr-only"
        >
          Remove
        </span>
      </a>
    </span>
  </li>
  <li>
    <span
      className="label label-info"
    >
      planet
      : 
      Earth
      <a
        href="#"
        onClick={[Function]}
      >
        <span
          aria-hidden="true"
          className="pficon pficon-close"
        />
        <span
          className="sr-only"
        >
          Remove
        </span>
      </a>
    </span>
  </li>
</ul>
`;
